app/assets/javascripts/kaui/kaui.js

```diff
@@ -261,3 +268,57 @@ function isBlank(value) {
         return false;
     }
 }
+
+// this function set popover for all tags that have class object-id-popover
+// attributes:
+//      data-id = content of the popover,object id; required
+//      title = title of the popover; not required
+//      id = (must be {{id}}-popover) used to close popover when the copy image is clicked; if present; if not present a timeout of 5s will apply; not required
+function setObjectIdPopover(){
+    $(".object-id-popover").each(function(idx, e){
+        $(this).popover('destroy');
+        $(this).off("shown.bs.popover");
+
+        $(this).popover({
+            html: true,
+            content: function() {
+                var template = '<div class="{{id}}-content" >' +
+                    '{{id}}&emsp;<i id="{{id}}-copy" class="fa fa-clipboard" aria-hidden="true"></i> ' +
+                    '<input id="{{id}}-placeholder" class="form-control hidden"> ' +
+                    '</div>';
+
+                var popover_html = Mustache.render( template , { id: $(this).data("id") });
+                return popover_html;
+            },
+            trigger: 'hover',
+            delay: { "show": 100, "hide": 4000 }
```

Comment:
Is there a way to also detect when a second popover is shown and hide the first one? Otherwise, the UI can get messy...

![image](https://user-images.githubusercontent.com/51940/40543610-03259794-5fda-11e8-91d3-b0845325668e.png)